### PR TITLE
Add Portuguese Translation file

### PR DESCRIPTION
**Pull request by tupaschoal**
Support Portugues translation for PrivateBin by adding its corresponding
file.

### CODE REVIEW

File: i18n/pt.json [hunk 1]
```diff
@@ -0,0 +1,142 @@
+{
+    "PrivateBin": "PrivateBin",
+    "%s is a minimalist, open source online pastebin where the server has zero knowledge of pasted data. Data is encrypted/decrypted <i>in the browser</i> using 256 bits AES. More information on the <a href=\"https://privatebin.info/\">project page</a>.":
+        "%s é um serviço minimalista e de código aberto do tipo \"pastebin\", em que o servidor tem zero conhecimento dos dados copiados. Os dados são cifrados e decifrados <i>no navegador</i> usando 256 bits AES. Mais informações na <a href=\"https://privatebin.info/\">página do projeto</a>.",
+    "Because ignorance is bliss":
+        "Porque a ignorância é uma benção",
+    "en": "pt",
+    "Paste does not exist, has expired or has been deleted.":
```
Comment (rugk) [inline i18n/pt.json]: translation missing
(this also creates the JSON format errors shown on GitHub in red)

File: i18n/pt.json [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+        "%s requer php 5.3.0 ou superior para funcionar. Desculpa.",
+    "%s requires configuration section [%s] to be present in configuration file.":
+        "%s requer que a seção de configuração [% s] esteja no arquivo de configuração.",
+    "Please wait %d seconds between each post.":
+        "Por favor espere %d segundos entre cada publicação.",
+    "Paste is limited to %s of encrypted data.":
+        "A cópia está limitada a %s de dados cifrados.",
+    "Invalid data.":
+        "Dados inválidos.",
+    "You are unlucky. Try again.":
```
Comment (rugk) [inline i18n/pt.json]: translation missing

File: i18n/pt.json [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+        "Erro ao salvar comentário. Desculpa.",
+    "Error saving paste. Sorry.":
+        "Erro ao salvar cópia. Desculpa.",
+    "Invalid paste ID.":
+        "ID de cópia inválido.",
+    "Paste is not of burn-after-reading type.":
+        "Cópia não é do tipo \"queime após ler\".",
+    "Wrong deletion token. Paste was not deleted.":
+        "Token de remoção inválido. A cópia não foi excluída.",
+    "Paste was properly deleted.":
```
Comment (rugk) [inline i18n/pt.json]: translation missing

File: i18n/pt.json [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+    "JavaScript is required for %s to work.<br />Sorry for the inconvenience.":
+        "JavaScript é necessário para que %s funcione.<br />Pedimos desculpas pela inconveniência.",
+    "%s requires a modern browser to work.":
+        "%s requer um navegador moderno para funcionar.",
+    "Still using Internet Explorer? Do yourself a favor, switch to a modern browser:":
```
Comment (rugk) [inline i18n/pt.json]: translation missing

File: i18n/pt.json [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+    "New":
+        "Novo",
+    "Send":
+        "Enviar",
+    "Clone":
+        "Clonar",
+    "Raw text":
+        "Texto sem formato",
+    "Expires":
```
Comment (rugk) [inline i18n/pt.json]: translation missing

File: i18n/pt.json [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+    "Burn after reading":
```
Comment (rugk) [inline i18n/pt.json]: translation missing

File: i18n/pt.json [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+        "Publicar comentário",
+    "Sending comment...":
+        "Enviando comentário...",
+    "Comment posted.":
+        "Comentário publicado.",
+    "Could not refresh display: %s":
+        "Não foi possível atualizar a tela: %s",
+    "unknown status":
+        "Estado desconhecido",
+    "server error or not responding":
```
Comment (rugk) [inline i18n/pt.json]: translation missing

File: i18n/pt.json [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+        "Enviando cópia (Por favor, mova o mouse para maior entropia)...",
+    "Sending paste...":
+        "Enviando cópia...",
+    "Your paste is <a id=\"pasteurl\" href=\"%s\">%s</a> <span id=\"copyhint\">(Hit [Ctrl]+[c] to copy)</span>":
+        "Sua cópia é <a id=\"pasteurl\" href=\"%s\">%s</a> <span id=\"copyhint\">(Pressione [Ctrl]+[c] para copiar)</span>",
+    "Delete data":
+        "Excluir dados",
+    "Could not create paste: %s":
+        "Não foi possível criar cópia: %s",
+    "Cannot decrypt paste: Decryption key missing in URL (Did you use a redirector or an URL shortener which strips part of the URL?)":
```
Comment (rugk) [inline i18n/pt.json]: translation missing

**Comment by tupaschoal** [comment]
I don't understand why some translations didn't get through, all I did was open the same unchanged file and commit again D:

Sorry!

**Comment by elrido** [comment]
To me too, I am just waiting for the JS refactoring to finish, so the changes in the i18n files can be merged.

**Comment by elrido** [comment]
Thanks again @tupaschoal, I merged it so this is not stuck forever. Will now re-merge master to the JS refactoring, so it is up-to-date there, too.

**Comment by tupaschoal** [comment]
Thank you!

**Pull request merged**